entity-framework/core/querying/related-data.md

```diff
@@ -82,21 +86,29 @@ INNER JOIN [Post] AS [p] ON [b].[BlogId] = [p].[BlogId]
 ORDER BY [b].[BlogId]
 ```
 
-While this avoids the performance issues associated with JOINs and cartesian explosion, it also has some drawbacks:
+> [!NOTE]
+> One-to-one related entities are always loaded via JOINs, as this has no performance impact.
+
+#### Enabling split queries globally
```

Comment:
Also document the warning user get if no default is set and query uses multiple includes.